### ISSUE TITLE
Same problem with dco on Linux @ordex

### ISSUE THREAD

**Issue by Katew212** [Katew212/Kate#2]
Same problem with dco on Linux @ordex 

I have attached userland and kernel logs.

[mullvad.log](https://github.com/OpenVPN/ovpn-dco-win/files/10696705/mullvad.log)
[dmesg.log](https://github.com/OpenVPN/ovpn-dco-win/files/10696706/dmesg.log)

_Originally posted by @lstipakov in https://github.com/OpenVPN/openvpn/issues/243#issuecomment-1424470979_

**Comment by schwabe** [comment]
Yes. It is the same problem. Please do not open duplicate issues.
